### DISCUSSION TITLE
Welcome to Arcticons Discussions!

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#333] [General]
Discussions! Whooo

**Comment by Aman9das** [comment]
Stuff left that i can think of:

- font
- navigation pane header-picture
- about header picture (i was thinking of an awesome collage of the contributors profile pictures..)
- wallpaper?

**Comment by Aman9das** [comment]
Hey  check this out!
https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=Donnnno/Arcticons

**Reply by Donnnno** [reply]
This is cool!

**Comment by Donnnno** [comment]
I think I'm gonna limit icon request to 10 icons per request for my own sanity. 🙃

**Comment by Donnnno** [comment]
One thing I noticed is that the GitHub download counter is going pretty fast ~10/20 downloads per day I think
